jit/v2/JIT_v2.0.0.yaml

```
@@ -563,6 +752,36 @@ paths:
                         property: carrierServiceCode
                         errorCodeText: mandatory property missing
                         errorCodeMessage: carrierServiceCode must be provided as part of a Terminal Call
+        '409':
+          description: |
+            In case creating a new or updating a **Terminal Call** linked to a **Port Call** that has been `OMITTED`, a `409` (Conflict) is returned.
+          headers:
+            API-Version:
+              $ref: '#/components/headers/API-Version'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                conflictRequestExample:
+                  summary: |
+                    Updating a Terminal Call that is OMITTED
+                  description: |
+                    Updating an OMITTED **Terminal Call** or creating a new **Terminal Call** linked to an OMITTED **Port Call**, returns a 409 (Conflict)
+
+                    **NB**: `errorCode` not yet standardized by DCSA. Value `7003` is just a "random example"
+                  value:
+                    httpMethod: PUT
+                    requestUri: /v2/terminal-calls/085a3207-5e45-49cf-8e1b-f8442beaf545
+                    statusCode: 409
+                    statusCodeText: Conflict
+                    statusCodeMessage: Trying to update a Terminal Call that has been OMITTED
```

```suggestion
                    statusCodeMessage: 'Trying to update a Terminal Call that has been OMITTED'
```
We should use quotes in String value examples, to make them validate and valid.